doc/source/getting_started/installing.rst

```diff
@@ -0,0 +1,461 @@
+We will provide instructions for this alternative.
+
+Another possibility is to use `virtualenv`_, which can be seen as a predecessor to Anaconda.
+Following the ``pip`` installation instructions in a ``virtualenv`` without the ``--user`` option works very well in our experience, but we do not provide explicit instructions for this variant.
+
+Which Python version to use?
+============================
+Any modern Python distribution supporting `NumPy`_ and `SciPy`_ should work for the core library, but some extensions require CPython (the standard Python distribution).
+
+ODL fully supports both Python 2 and Python 3.
+If you choose to use your system Python interpreter (the "pip install as user" variant), it may be a good idea to stick with the default one, i.e. the one invoked by ``python``.
+Otherwise, we recommend using Python 3, since Python 2 support will be discontinued in 2020.
+
+Development enviroment
+======================
+Since ODL is object-oriented, using an Integrated Development Environment (IDE) is recommended, but not required.
+The most popular ones are `Spyder`_ which works on all major platforms and can be installed through both ``conda`` and ``pip``, and `PyCharm`_ which can be integrated with any text editor of your choice, such as Emacs or Vim.
+
+
+------
+
+.. _alternative_1:
+
+***************************************************************************
+Alternative 1: Installing a release using ``conda`` (recommended for users)
+***************************************************************************
+
+TL;DR
+=====
+- Download and install `Miniconda`_
+- Create conda environment:
+
+  .. code-block:: bash
+
+    $ conda create -c odlgroup -n odl-py35 python=3.5 odl odlcuda matplotlib pytest scikit-image spyder
+
+Installing Anaconda
+===================
+Even though a Python interpreter is included by default in virtually all Linux distributions, it is advisable to use Anaconda's Python  ecosystem since it gives you full flexibility in the Python version you use and which packages you install.
+
+Download Anaconda from the Continuum Analytics home page.
+You may choose to download the `full Anaconda <https://www.continuum.io/downloads>`_ variant, but we recommend the slim `Miniconda`_ distribution since many of the packages included in full Anaconda are out of date anyway and need to be updated.
+Note that the choice of Python version (2 vs. 3) of the Anaconda installer is not very important since you can later choose to create conda environments with any Python version (see below).
+
+Make sure that during installation, your ``PATH`` variable is adapted such that ``conda`` and other scripts can be found by your shell::
+
+    Do you wish the installer to prepend the Miniconda3 install location
+    to PATH in your /home/user/.bashrc ? [yes|no]
+    [no] >>> yes
+
+After opening a new terminal (for the changed ``PATH`` variable to take effect), you can run
```

Comment:
I guess you are right. But perhaps go for "opening a new" to "restarting"? That kinda implies that `exec bash` is OK.